### ISSUE TITLE
[Future] Support real-time audio transcription

### ISSUE THREAD

**Issue by JackZhang1994** [azkadev/whisper_dart#11]
Hi thanks for the lib,

 i'm still a beginner in cpp. And I found [whisper.cpp](https://github.com/ggerganov/whisper.cpp/tree/master/examples/stream) support real-time audio transcription. Do you have any plans for support, or can you write an demo project, or can you help me make a .so file?
@azkadev @JeisonSanches 

Thanks

**Comment by azkadev** [comment]
Yes I will add the realtime feature but not right now because I'm a bit busy, for the demo I've updated the readme, and for compiling to shared libraries I've written here https://github.com/azkadev/whisper_dart/tree/main/native_lib

Thank you very much, sorry my english is bad

**Comment by JackZhang1994** [comment]
Thx

**Comment by Matzoh** [comment]
> Yes I will add the realtime feature but not right now because I'm a bit busy, for the demo I've updated the readme, and for compiling to shared libraries I've written here https://github.com/azkadev/whisper_dart/tree/main/native_lib是的，我会添加实时功能，但不是现在，因为我有点忙，对于演示我已经更新了自述文件，并且为了编译到我在这里编写的共享库 https://github.com/azkadev/ Whisper_dart/树/main/native_lib
> 
> Thank you very much, sorry my english is bad非常感谢，抱歉我的英语不好

Hope for early support
Thanks
